frontend/src/components/regions/UsersAndGroups/utils.js

```diff
@@ -0,0 +1,30 @@
+const PR_WARNING_AGE_THRESHOLD = 1
+const PR_ERROR_AGE_THRESHOLD = 2
+const TILDE_ERROR_AGE_THRESHOLD = 3
+const TILDE_WARNING_AGE_THRESHOLD = [1,2]
+
+export const timeDifferenceInDays = (time) => {
+  return Math.ceil(Math.abs(new Date() - new Date(time)) / (1000 * 60 * 60 * 24) - 1)
+}
+
+export const getPrStatus = (oldestOpenPrTime) => {
+  const ageInDays = timeDifferenceInDays(oldestOpenPrTime)
+  if (ageInDays >= PR_ERROR_AGE_THRESHOLD){
+    return "error";
+  }
+  if(ageInDays === PR_WARNING_AGE_THRESHOLD){
```

Comment:
Please dont use ===. Use >= only.

What if we want to change the configuration?

Imagine we decided that:
```
const PR_WARNING_AGE_THRESHOLD = 100
const PR_ERROR_AGE_THRESHOLD = 200
```

Your code should still work.  And your tests should still pass.